### PR TITLE
UI Branding and formating adjustments

### PR DESCRIPTION
**Pull request by Ryan-Cleminson**

### CODE REVIEW

**Comment by albert118** [comment]
Check out these links

https://carbondesignsystem.com/guidelines/typography/overview/

https://carbondesignsystem.com/guidelines/typography/type-sets/

File: src/frontend/src/styles/pages/Groups.scss [hunk 1]
```diff
@@ -42,6 +30,10 @@
     }
 }
 
+.groups-page__datatable{
+    padding-bottom: 5%
```
Comment (albert118) [inline src/frontend/src/styles/pages/Groups.scss:34]: ^

File: src/frontend/src/components/AppFooter/AppFooter.scss [hunk 2]
```diff
@@ -311,6 +311,7 @@
 }
 
 .acknowledgement__text p {
+    font-size: 0.8125rem;
```
Comment (albert118) [inline src/frontend/src/components/AppFooter/AppFooter.scss:314]: see the typography links I commented previously, these can be provided for us

**Review by albert118 — COMMENTED** [review]
Looks good to me, a few extra tweaks we can add in to clean up would be nice like we discussed,

- consistent typography
- better height distribution for the content (min-height)
- comments on some of the spacing choices
- shrink the footer a little maybe
- and drop the footer COO legal note
- drop dead buttons
- drop the extra menu link items (from the header)

**Comment by Ryan-Cleminson** [comment]
@albert118 this PR is redundant -> #107

**Pull request closed without merging**